portal-impl/src/com/liferay/portal/setup/SetupWizardUtil.java

```diff
@@ -193,7 +193,8 @@ public static void updateSetup(
 		_updateCompany(httpServletRequest, unicodeProperties);
 
 		_updateAdminUser(
-			httpServletRequest, httpServletResponse, unicodeProperties);
+			httpServletRequest, httpServletResponse, unicodeProperties,
+			databaseConfigured);
```

Comment:
@yuhai Here the variable name is pretty confusing. Actually `databaseConfigured == true` means the database is **NOT** changed by setup wizard. Could you add an SF commit before this commit to give it a more straightforward name, like "databaseConfigurationUnchanged" or something?